mozilla-release/toolkit/mozapps/installer/linux/rpm/mozilla.spec

```diff
@@ -20,11 +20,12 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
 #AutoProv:       no
 
-BuildRequires:  desktop-file-utils
+# CLIQZ: on ubuntu dependency has to be installed with apt-get
+#BuildRequires:  desktop-file-utils
```

Comment:
this rpm package specification. it's executed with rpmbuild tool that looks for dependencies in rpm package repository. on ubuntu this repository is empty so procedure fails even though system satisfies the requirement